packages/frontend/editor-ui/src/components/AssignmentCollection/Assignment.vue

```diff
@@ -86,6 +87,10 @@ const { resolvedExpressionString, isExpression } = useResolvedExpression({
 	additionalData: resolvedAdditionalExpressionData,
 });
 
+const expressionOutput = computed(() => {
+	return getResolvedExpression(isExpression.value, resolvedExpressionString.value);
```

Comment:
Could we add this behavior (null or '' if not an expression) to `resolvedExpressionString` from `useResolvedExpression` instead of adding a new util? I think most/all callsites will want this behavior